README-PNL.md

```diff
@@ -111,18 +112,31 @@ You can read the [Cookies](#cookies) section for details.
 
 Launch a DPdash instance as follows:
     
-    singularity run -B ${state}:/data -B ${data}:/project_data ${DPDASH_IMG} /sw/apps/dpdash/singularity/run.sh
+    singularity run \
+    -B ${state}:/data \
+    -B ${state}/dpdash/configs/dashboard:/sw/apps/dpdash/server/configs \
```

Comment:
For making `require`-->`import ../` type changes, if the two-step mounting is essential, you probably don't have a choice but I am nervous about asking a user to mount in two steps as opposed to just one. 

By the way, did you already include the new mounts in the convenience scripts? The PR is huge, I didn't get there yet.

Reply:
> The mounts are still all in one terminal command but I wanted to make it more readable by splitting up the lines in the docs with \ [newline]

I know, that's not the concern, this is:

> I do agree these mount statements are getting unwieldy.

Reply:
In that case, should we make the convenience scripts the preferred way of accessing these commands and make the manual bindings for advanced users? The mounts are required for the `import` statements.

Reply:
I can't think of a way for these two folders to be mounted in the same statement / from the same folder, because `configs` is pre-transpiled and `dist` is post-transpiled. The flow is like this:

1. Configs are placed in `server/configs`
2. `npm run transpile` transpiles ALL contents of `/server`, including `/configs`, and dumps the result into `/dist`
3. Code in `/dist` is run by Node

In other words, we need to mount a directory (`configs`) that only lives inside of `server` and a directory that only lives outside of `server` (`dist`), i.e. at the project root. If we mount both of them in one action, they'll either both or neither live inside of `server`.

Reply:
> but if you think that's worth it

Definitely, minimize complexity at all costs in software designing. I shall rethink your strategy and give it a go later.

Reply:
> wait for your further comment?

Yes

Reply:
No need to do any modification on this, thank you.